source/framework/core/src/startup.cpp

```diff
@@ -25,6 +25,10 @@ map<string, RESTVirtualConverter*> RESTConverterMethodBase = {};
 struct __REST_CONST_INIT {
    public:
     __REST_CONST_INIT() {
+// TODO: fix to avoid this dirty fix
```

Comment:
@nkx111 this is what I currently do to support testing, but its very ugly, perhaps you have some better idea?

With this "fix" if you build and install REST with testing enabled, perhaps it won't work as intended due to this line. It has no effect if testing is not enabled (default).

Reply:
How do you enable testing? Perhaps you should provide more details on the PR description so that we are able to reproduce what it is included at this PR